assignment-2/7s.md

```diff
@@ -1,2 +1,171 @@
+- They like cross-pollination instead of 'standarization', in order to spread knowledge, trends and tools between squads. It does not break the principal of 'issolated missions' because cross-pollination is used to spread the uses of tools like github for example. 
+- Their code is shared on an internal open source model. 
+- Squads make small and frequent releases as a routine, which is enable due decoupling their services. 
+- Their self-service model pretend minimizes the need for handoffs. 
+- They uses release trains and feature toggles aiming get things into production early and often.
+- They claim that their culture is based also on the people, therefore the focus on motivation, community and trust rather than structure and control.
+- Product development is based on Lean. They release their new features to a small group of customers limiting the 'blast radius' of a possible fail and it allows them to conduce a lot of small experiments. They report that it produces a lot of waste but this can be seen from a Lean perspective.
+They have something called 'core metrics data'. The data is feed dynamically due constant experimentation.
+
+##### Staffing
+
+> Staffing refers to people in the organization and how they are recruited, selected, and trained to
+> perform their jobs. Leadership is included in this category, whether leaders are selected from the
+> outside or developed from within. Staffing addresses the question of how the organization
+> assures it has the right people to do the work.
+
+Spotify claims to have a very clear definition of what and how employees careers should develop in order to achieve the most important organization goals.[6]"Everyone in *Spotify tech* should have a way to grow their careers and expand the impact of their -work no matter what role they play".
+The Spotify technology career framework, consist on letting the employee transfer itself on as many roles and domain expertise as possible in order to gain experience and later evaluate their impact into the organization.
+*On recognition and promotions* Spotify observes constantly the impact someone has at a wider scope, and they have a formal ways to promote their employees
+*On setting the step for a new employee* Spotify gives 6 months to all new employees to get involved with their role and the organization.
+*On compensation* A formal promotion should lead to higher pay in a straightforward and transparent way.
+
+##### Skills
+
+> Skills are the special, distinctive competencies of the organization, both in the people and the
+> organization itself. What is the organization good at, and what special skills and competencies
+> are found its people?
+
+What is the organization good at?
+Spotify seems to be good at keeping their organization dynamic, focusing on their community rather to an hierarchical structure.
```

Comment:
Statements with 'seems' are bad most of the time. Spotify is good, bad or somewhere in the middle. It is your task to analyze this and say something useful about it. If you eventually have hard numbers/logical conclusions and are still in doubt, only then you may say 'it seems that...'. Now there is no argumentation at all. Actually there are a couple of easy things to say where Spotify is very good at. Like...?